lib/modules/datasource/github-runners/index.spec.ts

```diff
@@ -3,18 +3,58 @@ import { GithubRunnersDatasource } from '.';
 
 describe('modules/datasource/github-runners/index', () => {
   describe('getReleases', () => {
-    it('returns releases if package is known', async () => {
+    it('returns releases for ubuntu', async () => {
       const res = await getPkgReleases({
         datasource: GithubRunnersDatasource.id,
         packageName: 'ubuntu',
       });
 
       expect(res).toMatchObject({
         releases: [
-          { version: '18.04' },
+          { version: '18.04', isDeprecated: true },
```

Comment:
The `github-runners` readme is outdated now that you're adding a flag for the beta version:

> This datasource returns a list of all _stable_ runners that are hosted by GitHub.
> This datasource ignores beta releases.
> The datasource is based on [GitHub's `runner-images` repository](https://github.com/actions/runner-images).
> 
> Examples: `windows-2019` / `ubuntu-22.04` / `macos-13`
> 
> ## Maintenance
> 
> New _stable_ runner versions must be added to the datasource with a pull request.
> Unstable runners are tagged as `[beta]` in the readme of the [`runner-images` repository](https://github.com/actions/runner-images).
> Once a runner version becomes stable, the `[beta]` tag is removed and the suffix `latest` is added to its YAML label.

Because we now:

- Return a list of stable and beta versions
- Flag beta releases instead of ignoring beta releases
- Add beta runner versions
- Should drop the beta flag once the version becomes stable

Can you update the `readme.md` file please?